src/Reflection/ReflectionClass.php

```diff
@@ -491,6 +495,54 @@ public function hasConstant(string $name) : bool
     }
 
     /**
+     * Get the reflection object of the specified class constant.
+     *
+     * Returns null if not specified.
+     *
+     * @param string $name
+     * @return ReflectionClassConstant|null
+     */
+    public function getReflectionConstant(string $name) : ?ReflectionClassConstant
+    {
+        return $this->getReflectionConstants()[$name] ?? null;
+    }
+
+    /**
+     * Returns array of constants for this class
+     *
+     * @param array $accumulator
+     * @param Node\Stmt $stmt
+     * @return ReflectionClassConstant[]
+     */
+    private function processReflectionConstants(array $accumulator, Node\Stmt $stmt) : array
+    {
+        if ($stmt instanceof ConstNode) {
+            $const = ReflectionClassConstant::createFromNode($this->reflector, $stmt, $this);
+            $accumulator[$const->getName()] = $const;
+        }
+
+        return $accumulator;
+    }
+
+    /**
+     * Get an array reflection object of the defined constants in this class.
+     *
+     * @return ReflectionClassConstant[]
+     */
+    public function getReflectionConstants() : array
+    {
+        if (null !== $this->cachedReflectionConstants) {
+            return $this->cachedReflectionConstants;
+        }
+
+        return $this->cachedReflectionConstants = array_reduce(
+            $this->node->stmts,
+            [$this, 'processReflectionConstants'],
```

Comment:
This method can just be inlined here I reckon, it's only used the once and I can't see it being re-usable?